src/main/java/com/fcgl/madrid/forum/controller/CommentController.java

```diff
@@ -18,9 +21,16 @@
     public void setCommentService(CommentService commentService) {
         this.commentService = commentService;
     }
+
     @PostMapping(path = "/postComment")
     public ResponseEntity<InternalStatus> postComment(CommentRequest commentRequest) {
         return commentService.postComment(commentRequest);
     }
 
+    @GetMapping(path = "/postComment")
```

Comment:
shouldnt this be getComment? it can cause confusion with 2 postComment methods and the it will also do what it says

Reply:
Usually convention is you name the `get` and `post` the same thing and then differentiate what they do with the `get` and `post` but I guess we didn't enforce that in this repo so I agree that it is confusing.

Example: 
endpoint:`/comment`
POST: '/comment': Adds a comment
GET: '/comment': Gets the comment

I'll switch to `getComment` to avoid confusion

Reply:
Done: https://github.com/fcgl/forum/pull/9/commits/72ff580ad9ff028694a24ad5ba513cd39c978332

Switched `postComment` to `getComment`